crates/sys/src/notification.rs

```diff
@@ -0,0 +1,95 @@
+//! Low-level bindings to Notificcation API.
+
+use core::ffi::c_char;
+use crate::{opaque, c_string};
+
+pub const RECORD_NOTIFICATION: *const c_char = c_string!("notification");
+
+opaque!(Notification);
+opaque!(NotificationSequence);
+
+extern "C" {
+    #[link_name="notification_message"]
+    pub fn message(notifications_app: *mut Notification, sequence: *const NotificationSequence);
+}
+
+pub mod vibro {
+    use super::NotificationSequence;
+
+    extern {
+        #[link_name="sequence_set_vibro_on"]
+        pub static ON: NotificationSequence;
+        #[link_name="sequence_reset_vibro"]
+        pub static OFF: NotificationSequence;
+    }
+}
+
+pub mod led {
+    use super::NotificationSequence;
+
+    pub mod red {
+        use super::NotificationSequence;
```

Comment:
You might be able to avoid some of the `use super::NotificationSequence` by using `use super::super::*`. 🌟

Reply:
Done.